### PR TITLE
migrate download and ingest into job manager

### PR DESCRIPTION
**Pull request by cooper-lzy**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/4.job-statements.md [hunk 1]
```diff
@@ -89,6 +89,32 @@ nebula> SUBMIT JOB STATS;
...
+`SUBMIT JOB DOWNLOAD HDFS`和`SUBMIT JOB INGEST`命令用于将 SST 文件导入 Nebula Graph。详情参见[导入 SST 文件数据](../nebula-exchange/use-exchange/ex-ug-import-from-sst.md)。
+
+`SUBMIT JOB DOWNLOAD HDFS`语句会下载指定的 HDFS 上的 SST 文件。
+
+`SUBMIT JOB INGEST`语句会将下载的 SST 文件导入图空间。
+
+示例：
+
+```ngql
+nebula> SUBMIT JOB DOWNLOAD HDFS "hdfs://192.168.10.100:9000/tmp";
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/4.job-statements.md:103]: nebula> SUBMIT JOB DOWNLOAD HDFS "hdfs://192.168.10.100:9000/tmp";
tmp-> 1.sst

Comment (whitewum) [inline docs-2.0/3.ngql-guide/4.job-statements.md:103]: Concurrently download example?

is `/dir/path/*` avaiable?

**Review by whitewum — CHANGES_REQUESTED** [review]

File: docs-2.0/3.ngql-guide/4.job-statements.md (hunk 1 above)
Comment (cooper-lzy) [reply docs-2.0/3.ngql-guide/4.job-statements.md:103]: tmp is a directory that contains all the`.sst` files.

File: docs-2.0/3.ngql-guide/4.job-statements.md [hunk 2]
```diff
@@ -89,6 +89,32 @@ nebula> SUBMIT JOB STATS;
...
+`SUBMIT JOB DOWNLOAD HDFS`和`SUBMIT JOB INGEST`命令用于将 SST 文件导入 Nebula Graph。详情参见[导入 SST 文件数据](../nebula-exchange/use-exchange/ex-ug-import-from-sst.md)。
+
+`SUBMIT JOB DOWNLOAD HDFS`语句会下载指定的 HDFS 上的 SST 文件。
+
+`SUBMIT JOB INGEST`语句会将下载的 SST 文件导入图空间。
+
+示例：
+
+```ngql
+nebula> SUBMIT JOB DOWNLOAD HDFS "hdfs://192.168.10.100:9000/sst";
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/4.job-statements.md:103]: 1.sst?

Comment (whitewum) [reply docs-2.0/3.ngql-guide/4.job-statements.md:103]: & multiple sstfiles?

**Review by randomJoe211 — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**